frontend/lib/src/components/widgets/DataFrame/DataFrame.test.tsx

```diff
@@ -70,50 +67,25 @@ describe("DataFrame widget", () => {
   })
 
   it("renders without crashing", () => {
-    const wrapper = mount(<DataFrame {...props} />)
-    expect(wrapper.find(GlideDataEditor).length).toBe(1)
-  })
-
-  it("should have correct className", () => {
-    const wrapper = mount(<DataFrame {...props} />)
-    expect(wrapper.find(StyledResizableContainer).prop("className")).toContain(
-      "stDataFrame"
-    )
+    render(<DataFrame {...props} />)
+    expect(screen.getAllByTestId("stDataFrameResizable").length).toBe(1)
   })
 
   it("grid container should use full width when useContainerWidth is used", () => {
-    const wrapper = mount(
-      <DataFrame {...getProps(new Quiver({ data: TEN_BY_TEN }), true)} />
+    render(<DataFrame {...getProps(new Quiver({ data: TEN_BY_TEN }), true)} />)
+    const dfStyle = getComputedStyle(
+      screen.getByTestId("stDataFrameResizable")
     )
-    const dataFrameContainer = wrapper.find(Resizable).props() as any
-    expect(dataFrameContainer.size.width).toBe(700)
-    expect(dataFrameContainer.size.height).toBe(400)
+    expect(dfStyle.width).toBe("700px")
+    expect(dfStyle.height).toBe("400px")
   })
 
   it("grid container should render with specific size", () => {
-    const wrapper = mount(<DataFrame {...props} />)
-    const dataFrameContainer = wrapper.find(Resizable).props() as any
-    expect(dataFrameContainer.size.width).toBe(400)
-    expect(dataFrameContainer.size.height).toBe(400)
-  })
-
-  it("Touch detection correctly deactivates some features", () => {
-    // Set window.matchMedia to simulate a touch device
-    window.matchMedia = jest.fn().mockImplementation(() => ({
-      matches: true,
-    }))
-
-    const wrapper = mount(
-      <DataFrame
-        {...getProps(
-          new Quiver({ data: TEN_BY_TEN }),
-          true,
-          ArrowProto.EditingMode.FIXED
-        )}
-      />
```

Comment:
Makes sense that direct prop testing on the component isn't viable anymore, but are there testable characteristics/attributes in the rendered DOM elements that we can test instead?
Ex: when GlideDataEditor's `fillHandle` is false, what happens in the component? Can you check a specific child element does not render anymore, etc

Reply:
Hmm I don't think there i sany testable characteristics / attributes in the rendered DOM elements. @LukasMasuch , can you confirm?

Reply:
Yep, indeed. These aspects are only detectable within the Canvas and not the DOM. But in this case, wouldn't it be possible to mock the `GlideDataEditor` and check if it has been called with the expected parameters? As it is described here: https://stackoverflow.com/a/72450016

Reply:
Will check this out later today. thanks!

Reply:
finished. thanks!

Reply:
Awesome, so for cases where DOM testing isn't possible, this is a good workaround with RTL 👍